shared/pinentry/index.render.desktop.js

```diff
@@ -57,7 +57,13 @@ export default class PinentryRender extends Component<DefaultProps, Props, State
     const submitPassphrase = () => this.props.onSubmit(this.state.passphrase, this.state.features)
 
     const isPaperKey = this.props.type === PassphraseCommonPassphraseType.paperKey
-    const typeStyle = {
+    const typeStyle: $Shape<{
```

Comment:
This is a case where flow is really tricky.

The type checks passed because flow didn't correctly infer this type.

so when we did `...typeStyle` below, even if onChange was wrong, flow wouldn't mind because it figured the `...typeStyle` will fix everything later.